src/NodeRegistry.sol

```diff
@@ -63,18 +59,15 @@ contract NodeRegistry is INodeRegistry, AccessControlDefaultAdminRules, ERC721 {
     /// @dev Mapping of token ID to Node.
     mapping(uint256 => Node) internal _nodes;
 
-    /// @dev Nodes with API enabled.
-    EnumerableSet.UintSet internal _activeApiNodes;
-
-    /// @dev Nodes with replication enabled.
-    EnumerableSet.UintSet internal _activeReplicationNodes;
+    /// @dev Nodes part of the canonical network.
+    EnumerableSet.UintSet internal _canonicalNetworkNodes;
 
-    /// @inheritdoc INodeRegistry
+    /// @dev The commission percentage that the node operator receives.
```

Comment:
AS this is a public variable, it is in the interface, where it's Natspec is defined.

Reply:
As far as I know only functions are defined in the interface. You'll get a warning if defining a public var in an interface, so I'm not sure if I follow.